### ISSUE TITLE
ImagickDraw::setTextUnderColor : colors are slightly different

### ISSUE THREAD

**Issue by dongjin84229** [Danack/Imagick-demos#46]
Reported from ImagickDraw::setTextUnderColor

The color of the text under color parameter and the color of the actual created picture are slightly different.

![image](https://user-images.githubusercontent.com/40761862/107394487-e9367b80-6b36-11eb-8da2-ea4fcdc7de21.png)
![image](https://user-images.githubusercontent.com/40761862/107394642-0f5c1b80-6b37-11eb-89d0-2ba71d6ccacf.png)

**Comment by Danack** [comment]
Well, that's weird.

Setting `rgb(255, 0, 128)` is also wrong. The color contains some green....

I don't think this is going to be my fault, it will be some weirdness in ImageMagick. 

Can you check what the behaviour is calling the similar function from the command line version of ImageMagick as per the examples given at: https://legacy.imagemagick.org/Usage/text/#undercolor

If that has the same behaviour, that's the best way of demonstrating an issue like this to them.

**Comment by Danack** [comment]
oh, and the other thing that should be tried is explicitly setting a colorspace.....something could be defaulting to CMYK, and then being converted oddly.

**Comment by dongjin84229** [comment]
Yes this is not your fault. As I used this function, there was a difference in color.
Is there any way to solve this?

**Comment by Danack** [comment]
> Is there any way to solve this?

I don't know and someone will need to investigate. 

Can you check what the behaviour is calling the similar function from the command line version of ImageMagick as per the examples given at: https://legacy.imagemagick.org/Usage/text/#undercolor
